app/pages/intl/Title.tsx

```diff
@@ -0,0 +1,38 @@
+import type { FunctionComponent } from 'react'
+import type { PropsWithTranslation, Translation, Translations } from '../../hocs/withTranslation'
+import { withTranslation } from '../../hocs/withTranslation'
+import { Language } from '../../hooks/useLanguage'
+
+interface TitleTranslation extends Translation {
+  welcome: string,
+  goodToSeeYou: string,
+  page: (page: number) => string
+}
+
+interface TitleProps {
+  name: string
+}
+
+// Simple Title component to demonstrate some translations
+const Title: FunctionComponent<PropsWithTranslation<TitleTranslation, TitleProps>> = (props) => {
+  return <p
+    className={'rounded bg-gray-800 text-gray-200 p-1 px-2'}
+  >{props.translation.welcome}! {props.translation.goodToSeeYou}, <span
+    className={'text-green-300'}
+  >{props.name}</span>. {props.translation.page(123)}</p>
+}
+
+const defaultTitleTranslations: Translations<TitleTranslation> = {
+  [Language.EN]: {
+    welcome: 'Welcome',
+    goodToSeeYou: 'Good to see you',
+    page: (page) => `Page ${page}`
+  },
+  [Language.DE]: {
+    welcome: 'Willkommen',
+    goodToSeeYou: 'Schön dich zu sehen',
+    page: (page) => `Seite ${page}`
+  }
+}
+
+export default withTranslation(Title, defaultTitleTranslations)
```

Comment:
This file should not be in the `pages` folder, but rather in a `components` folder!
`/intl/Title` shouldn't be a navigable route.

Reply:
Its just an example. _Moved!_